### ISSUE TITLE
node-pathwatcher error undefined symbol _Z12PlatformInitv

### ISSUE THREAD

**Issue by TomFreudenberg** [atom/node-pathwatcher#94]
Trying to compile and run node-pathwatcher on FreeBSD. Compiling runs fine without any issue but when using that package

`require 'pathwatcher'`

error ist thrown 

```
node_modules/pathwatcher/build/Release/pathwatcher.node: Undefined symbol "_Z12PlatformInitv"
```

Thanks for help

**Comment by TomFreudenberg** [comment]
I added also a question on stackexchange:

http://stackoverflow.com/questions/33480307/freebsd-cannot-use-pathwatcher-npm

**Comment by TomFreudenberg** [comment]
It depends on missing src for FreeBSD

I created a PR #95 to fix this

**Comment by 50Wliu** [comment]
Fixed by #95.
